src/components/cells/TimeSliderCell.vue

```diff
@@ -10,7 +10,7 @@
         :interval="0.25"
         :lazy="true"
         :min="0"
-        :max="maxHoursByDay"
+        :max="12"
```

Comment:
Why change this max value?
A bigger "hoursByDay" value can be selectable with the button on the right of slider (see line Line 25)

Reply:
It's because it breaks the styling.